src/main/java/itAcademy/ORM/mapping/util/Util.java

```diff
@@ -0,0 +1,127 @@
+package itAcademy.ORM.mapping.util;
+
+import itAcademy.ORM.connection.BaseDataSourceFactory;
+import itAcademy.ORM.connection.DataSourceFactory;
+import itAcademy.ORM.mapping.Field;
+import itAcademy.ORM.mapping.Table;
+import itAcademy.ORM.reflection.ReflectionAPI;
+
+import java.lang.reflect.Type;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import java.util.Map;
+
+public class Util {
+
+
+    private static Connection connection;
+
+    static {
```

Comment:
@YuliaDolenko , will remove this static block. Use DaoSupport class for creating the connection to DB. Looking for example at https://github.com/HannaHurieva/IT_ACADEMY_ORM/blob/transaction/src/main/java/itAcademy/ORM/Demo.java
Or you will need to merge the "dev" branch after changing it from @dyominov .